src/components/project/user/list.vue

```diff
@@ -152,6 +142,15 @@ export default {
         : this.$t('common.noResults');
     }
   },
+  watch: {
+    searchTerm(searchTerm) {
+      if (!searchTerm) {
+        this.clearSearch();
+      } else {
+        this.search(true);
```

Comment:
I don't think `this.search()` takes an argument.

```suggestion
        this.search();
```